src/Raven.Server/Config/Categories/ConfigurationCategory.cs

```diff
@@ -109,6 +109,8 @@ public void Initialize(Func<string, SettingValue> getSetting, string serverDataD
                     if (value == null)
                         continue;
 
+                    value = value.Trim();
+
```

Comment:
Don't we want to handle this for `foo ; bar ; `, so we'll need to handle this for `string[]` as well.
Also, probably need to add `RemoveEmptyEntries` there, too